### DISCUSSION TITLE
How to get the Laue diffraction using nCrystal?

### DISCUSSION THREAD

**Discussion by iyadalqasir** [mctools/ncrystal#148] [Q&A] [answered]
Hi All,
Is nCrystal capable of getting the Laue diffraction pattern using? if yes, how can we do that?

Thank you
Iyad

**Comment by marquezj** [comment, marked as answer]
Hi @iyadalqasir. Yes, you can. You need to define the scattering object as a single crystal (with an specific orientation and mosaicity), sample the scattering object with the beam direction and energy/wavelength, and plot the results.

[This Google Colab Notebook](https://colab.research.google.com/drive/1wAq2dErn017GSPp3mNGTjEZZNUwmpnE5?usp=sharing) is a simple example. If you want to modify it, go to [File] -> [Save a copy in Drive] and it will open a window with your own copy.

(Maybe someone can propose better orientations or wavelengths to plot)

Cheers.

**Comment by iyadalqasir** [comment]
Thank you, Ignacio.
Are you planning to attend the coming CSEWG meeting? I will give a talk
about graphite via Teams.

Kindest Regards,
*Iyad Al-Qasir, PhD*
